src/table/head/table-head-cell.component.ts

```diff
@@ -17,7 +17,7 @@ import { TableHeaderItem } from "./../table-header-item.class";
 	template: `
 		<button
 			class="bx--table-sort"
-			*ngIf="!skeleton && this.sort.observers.length > 0 && column.sortable"
+			*ngIf="sortable && !skeleton && this.sort.observers.length > 0 && column.sortable"
```

Comment:
I _think_ we can remove `this.sort.observers.length > 0` ... I'm not sure how it's even working in the template as is ...

Reply:
Should it be removed in the other places it is present as well?

Reply:
Basically just checks to see if there's any listeners to the observer ... it seems to be safe to remove in the head cell, but I'm not sure about the other components